ember/ember-cli-build.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```

Comment:
do we have to add the `enabled` key here or will it default to that automatically?